src/test/java/com/yahoo/bullet/querying/RunningQueryTest.java

```diff
@@ -70,5 +70,23 @@ public void testStartTime() {
         long end = System.currentTimeMillis();
         Assert.assertTrue(runningQuery.getStartTime() >= start);
         Assert.assertTrue(runningQuery.getStartTime() <= end);
+        Assert.assertFalse(runningQuery.isTimedOut());
+    }
+
+    @Test
+    public void testTimingOut() throws Exception {
```

Comment:
Mock System.currentTimeMillis() and get rid of sleep if not too difficult? This test may fail due to system clock problem.

Reply:
I don't want to use things like Powermock to mock static methods if possible. Any other suggestions? I agree it may fail. By the way, there are multiple tests that use this kind of sleep in the Tumbling window tests.

Reply:
I don't have better suggestions. Then just leave it as.